packages/components/src/components/header/header.styles.ts

```diff
@@ -0,0 +1,83 @@
+import { css } from 'lit';
+
+export default css`
+  :host {
+    display: block;
+  }
+
+  .header {
+    background: var(--syn-color-neutral-0);
+    box-shadow: inset 0 -1px 0 0 var(--syn-color-neutral-400);
```

Comment:
Hrhr. :D Is this the border-bottom inside the element?

Reply:
TBH I'm not sure if is correct, as this changes the paddings, so that we have 1px less to the bottom. Do you see a problem with border-bottom?

Reply:
Just had a talk with @mariohamann, we leave it like this because it will make styling easier in the end (e.g. when using the to be created horizontal navigation, we will be able to slightly override the shadow, but it would be harder when using a border).